en/docs/administration/customize-the-developer-portal.md

```diff
@@ -0,0 +1,36 @@
+# Customize the Developer Portal
+
+When you design APIs and publish them, you may want to present them in a unique theme specific to your organization or brand. Therefore, Choreo allows you to customize the Developer Portal theme.
+
+To do this, you can access the default Developer Portal theme configuration by following the steps given below:
+
+1. Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google/ GitHub/ Microsoft account.
+
+2. If you are a new user, create an organization. Enter a unique organization name. For example, "Stark Industries".
+
+    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+3. In the left pane, click **Settings**.
+
+4. In the **Organization** tab, click **Devportal Theme**.
+
+    ![Access Devportal theme](../assets/img/administration/devportal-theme/access-devportal-theme.png)
+
+    Once you access the theme, you can customize the Home page, color theme, font, header and footer, logos, etc., by expanding the **General**, **Home Page**, **Color Palette**, **Typography**, and **Footer Links** sections.
+
+5. Make a change to the theme. For example, let's update the title text on the home page and change the color theme. 
+
+    1. To update the test on the Home page, expand the **Home Page** section, and in the **Title** field, change the default text (for example, to `Try our APIs!`).
+   
+    2. To update the color theme, expand the **Color Palette** section, and change the colors as required.
+   
+    3. Click **Preview** to view a preview of the Developer Portal with the changes you made. For example, let's assume you changed the primary color of the background to `#C3C5CD` and the primary color of the buttons to `#086634`. Based on these changes, the preview appears as follows.
+
+         ![Preview of customization](../assets/img/administration/devportal-theme/preview-of-customization.png)
+   
+    4. Click **Save**.
+
+6. Sign in to Choreo Developer Portal at https://devportal.choreo.dev.
+
+The Home page will appear as it did in the preview.
+
```

Comment:
Can someone reset the theme? If so shall we mention that as well?

Reply:
Added a new section to reset the theme.